armi/bookkeeping/visualization/tests/test_vtk.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2020 TerraPower, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import numpy
+from pyevtk.vtk import VtkTetra
+
+from armi.reactor.tests import test_reactors
+from armi.reactor import components
+from armi.reactor import blocks
+from armi import settings
+from armi.bookkeeping.visualization import vtk
+
+
+class TestVtkMesh(unittest.TestCase):
```

Comment:
quick desc of what this tests useful but not required. is this basically a test of the vtk system?